package.json

```diff
@@ -10,17 +10,21 @@
     "format": "prettier --write './**/*.{js,jsx,ts,tsx,css,md,json}' --config ./.prettierrc"
   },
   "dependencies": {
+    "@apollo/client": "^3.4.1",
     "@material-ui/core": "^4.12.2",
     "@material-ui/icons": "^4.11.2",
     "@material-ui/lab": "*",
     "@reactchartjs/react-chart.js": "^1.0.0-rc.3",
+    "apollo-server-micro": "^3.1.2",
     "aws-sdk": "^2.932.0",
     "axios": "^0.21.1",
     "bcrypt": "^5.0.1",
     "chart.js": "^2.9.4",
     "classnames": "^2.2.6",
     "express-validator": "^6.12.0",
+    "graphql": "^15.5.1",
     "lodash": "^4.17.21",
+    "micro": "^9.3.4",
```

Comment:
I actually just did a quick search and for our needs, I don't think we need this package. Unless you have an article that says otherwise, maybe `npm uninstall micro`